blocklist.sh

```diff
@@ -0,0 +1,22 @@
+# SPDX-FileCopyrightText: (c) 2024 Ring Zero Desenvolvimento de Software LTDA
+# SPDX-License-Identifier: GPL-2.0-only
+#!/bin/sh
```

Comment:
must be in the first line, iirc.

Reply:
it should.. ;-).. you're right anyway.. my bad..